PSScripts/Set-ArmParameters.ps1

```diff
@@ -4,6 +4,4 @@ param(
     [string]$Environment
 )
 
-$ApimExists = $null -ne (Get-AzureRmApiManagement -ResourceGroupName "dfc-$Environment-shared-rg" -Name "dfc-$Environment-shared-apim" -ErrorAction Ignore)
-Write-Verbose "Writing value $ApimExists to variable apimExists"
-Write-Output "##vso[task.setvariable variable=apimExists]$($ApimExists.ToString().ToLower())"
\ No newline at end of file
+$env:ApimExists = $null -ne (Get-AzureRmApiManagement -ResourceGroupName "dfc-$Environment-shared-rg" -Name "dfc-$Environment-shared-apim" -ErrorAction Ignore)
```

Comment:
If you use $env to set the variable rather than writing it to task.setvariable won't the variable only be accessible in the task that runs the script?  I thought you had to use task.setvariable to use the variable other tasks?

Reply:
Craig said not.